src/views/AcceptTransfer.tsx

```diff
@@ -53,7 +53,7 @@ function useAcceptTransfer() {
     ),
     useCallback(
       () => Promise.all([syncExtras(_point), syncControlledPoints()]),
-      [_point, syncControlledPoints, syncExtras]
+      [_point]
```

Comment:
Having `syncControlledPoints` and `syncExtras` as dependencies causes the crazy amount of HTTP requests, again because of reference equality checks.